source/fundamentals/builders/builders-data-classes.txt

```diff
@@ -49,6 +49,12 @@ To implement this functionality, you must add the
 ``mongodb-driver-kotlin-extensions`` dependency to your dependencies
 list.
 
+.. sharedinclude:: dbx/jvm/BOM-installation-tip.rst
+
+   .. replacement:: installation-guide
+
+      :ref:`Installation Guide<kotlin-get-started-install-bom>`
```

Comment:
S: nice work implementing the shared include!
A few suggestions:
1. Based on the wording in the sharedinclude, you should include a period here
2. The style guide recommends being super clear about link targets, so make the link text more clear

```suggestion
.. sharedinclude:: dbx/jvm/BOM-installation-tip.rst

   .. replacement:: installation-guide

      the :ref:`Add the Driver Bill of Materials <kotlin-get-started-install-bom>`
      step of the Quick Start guide.
```

Reply:
This applies to other includes in this PR